src/utils/consistency/index.rs

```diff
@@ -1,11 +1,12 @@
+use rayon::iter::ParallelIterator;
 use super::data::{Crate, CrateName, Data, Release, Version};
 use crate::Index;
 
 pub(crate) fn load(index: &Index) -> Result<Data, anyhow::Error> {
     let mut data = Data::default();
 
-    index.crates()?.walk(|krate| {
-        data.crates.insert(
+    data.crates = index.crates()?.crates_parallel().map(|krate| {
```

Comment:
Lol, I had written a comment that this could be `let crates = ...; Ok(Data { crates })`; but changed my mind. Looks like Clippy would prefer that though.